composer.json

```diff
@@ -100,6 +100,7 @@
         "drupal/coder": "^8.3",
         "drupal/devel": "^5.1.2",
         "drupal/single_content_sync": "^1.3",
-        "drush/drush": "^12.1"
+        "drush/drush": "^12.1",
+        "phpunit/phpunit": "^9.6"
```

Comment:
Is there a point to using --dev at this point? I think it's useful to have things like phpunit and coder not added in production but not sure the flag is still the recommended way of excluding them?

Reply:
Separating them in composer is still recommended, but you don't use the `--no-dev` flag when installing anymore. Instead you set a `COMPOSER_NO_DEV` environment variable: https://getcomposer.org/doc/03-cli.md#composer-no-dev